packages/ui/Scrollable/Scrollable.js

```diff
@@ -604,8 +604,11 @@ class ScrollableBase extends Component {
 	getRtlX = (x) => (this.props.rtl ? -x : x)
 
 	onMouseDown = (ev) => {
-		this.stop();
-		forward('onMouseDown', ev, this.props);
+		if (this.props.onMouseDown) {
```

Comment:
`forward` doesn't need to be gated like this so we could simplify this a bit.  Perhaps a better approach is to use `forwardWithPrevent()` and call `preventDefault` inside `moonstone/Scrollable`?